src/ray/constants.h

```diff
@@ -0,0 +1,12 @@
+#ifndef RAY_CONSTANTS_H_
+#define RAY_CONSTANTS_H_
+
+/// Length of Ray IDs in bytes.
+constexpr int64_t kUniqueIDSize = 20;
+
+/// Prefix for the object table keys in redis.
+constexpr char kObjectTablePrefix[] = "ObjectTable";
+/// Prefix for the task table keys in redis.
+constexpr char kTaskTablePrefix[] = "TaskTable";
```

Comment:
These prefixes aren't used yet, right? We're still hardcoding `TT` and `OI` and `OL` and things like that in various places in the code so at some point we'll have to replace those with these constants.

Reply:
Yeah, that's right!

Reply:
I think a more "correct" way (at least how I was educated at my previous employer :p) of defining constants are,
* In header: `extern const int64_t kUniqueIDSize; ...; extern const char* const kObjectTablePrefix;`
* In `.cc`: `const int64_t kUniqueIDSize = 20; ...; const char* const kObjectTablePrefix = "ObjectTable"`

Just an FYI, no need to do it this way in this PR.